### ISSUE TITLE
Support `OAuth2JWTBearer` auth type.

### ISSUE THREAD

**Issue by jjtang1985** [SAP/cloud-sdk-js#1437]
This feature request is created on behalf of a user who contacted us.
This auth type is not supported by the SDK for the time being, but this auth type has to be used for a specific service.

**Comment by navinkrishnan** [comment]
Hi @jjtang1985 ,

Can you give me the canary version number this is supported ?
I took `1.46.1-20210713150118.0` from the public npm and still see the message

```
'Error during request to remote service: Failed to build HTTP request for destination: failed to build headers! Caused by: The destination used "OAuth2JWTBearer" as authentication type which is not supported by the SDK.
```

Should I look for a different version in the internal nexus ?

Regards,
Navin

**Comment by jjtang1985** [comment]
Hi @navinkrishnan ,

The version you used from the public npm should contain the new auth type.
Please only use the public npm for the `canary` version of the SDK core to avoid potential delays.
It would be even better, if you use the `1.46.1-20210713150118.0` instead of `canary`, as the `canary` might change.

Locally, you can open the file:
```
node_modules/@sap-cloud-sdk/core/dist/connectivity/scp-cf/authorization-header.js
```
and search for `OAuth2JWTBearer`, which is added.

For CF deployment, you can push the `package-lock.json` file (double check the SDK core version) and `npm ci` should give you the same SDK core version as local deployment.

Please let me know if it works.

Best regards,
Junjie

**Comment by jjtang1985** [comment]
Hi @navinkrishnan ,

we have released version [1.47.0](https://github.com/SAP/cloud-sdk-js/releases/tag/v1.47.0), please use the latest version.

I'll close the ticket.
Please reopen in case you have questions.

Best regards,
Junjie

**Comment by navinkrishnan** [comment]
Hi @jjtang1985 

As mentioned in the mail, this is tested and it is working for me. I'm able to successfully consume the SAP Graph APIs via the CAP application.

Regards,
Navin
